build.gradle

```diff
@@ -35,12 +35,12 @@ ext {
             node : '8.7.0',
 
             // Main Framework
-            dbflute : '1.2.1',
+            dbflute : '1.2.3',
             h2 : '1.4.199',
-            lastaflute : '1.1.6',
+            lastaflute : '1.1.8',
```

Comment:
Please use LastaFlute-1.2.0 that will be released today!(^^

Reply:
Upgraded 51b160108b6457ce33899df25132c86540d96726